src/store/houses/actions.ts

```diff
@@ -12,4 +12,15 @@ export const actions: ActionTree<HousesStateInterface, RootStateInterface> & Act
   [HousesActionsEnum.SET_MODAL_DISPLAY]: ({ commit }, payload) => {
     commit(HousesMutationsEnum.SET_MODAL_DISPLAY, payload);
   },
+  [HousesActionsEnum.ADD_HOUSE]: async ({ commit }, payload) => {
+    try {
+      const { data } = await HTTP.post(`/cooperations/${payload.cooperationId}/houses`, payload);
+      commit(HousesMutationsEnum.ADD_HOUSE, data);
+    } catch (e: any) {
+      console.log(e.response);
+    }
+  },
+  [HousesActionsEnum.SET_ADD_HOUSE_MODAL]: ({ commit }, payload) => {
+    commit(HousesMutationsEnum.SET_ADD_HOUSE_MODAL, payload);
+  },
```

Comment:
You can remove this action, cause it wont make some server connected or async logic. You can call mutation directly from the component

Reply:
I did it by analogy, but you`re right it would be better called from he component. I just wanted to safe right flow kinda action => mutation, however I forgot about the main action aim - async logic)